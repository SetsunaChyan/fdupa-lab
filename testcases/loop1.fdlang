function main(){
x = 1;

while (x < 10) {
    y = y + x;
    x = x + 1;
    check_interval(x, 0, 9);
}

check_interval(y, 45, 45);
}