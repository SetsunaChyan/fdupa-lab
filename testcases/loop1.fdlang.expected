Line 7:  NO; x in [2, 10]
Line 10: YES; y in [45, 45]
